core/src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/BasicLoadBalancingPolicy.java

```diff
@@ -288,14 +292,21 @@ protected Set<Node> getReplicas(@Nullable Request request, @Nullable Session ses
     if (!maybeTokenMap.isPresent()) {
       return Collections.emptySet();
     }
+    TabletMap tabletMap = context.getMetadataManager().getMetadata().getTabletMap();
 
     // Note: we're on the hot path and the getXxx methods are potentially more than simple getters,
     // so we only call each method when strictly necessary (which is why the code below looks a bit
     // weird).
     CqlIdentifier keyspace;
+    CqlIdentifier table = null;
     Token token;
     ByteBuffer key;
     Partitioner partitioner = null;
+
+    if (request instanceof BoundStatement) {
+      table = ((BoundStatement) request).getPreparedStatement().getTable();
+    }
```

Comment:
I think right now yes. `PreparedStatement` by itself won't appear here because it does not subclass `Request`. I am not sure if there are other types of `Request`s i should be mindful of with Tablets.